src/chocolatey/infrastructure/logging/Log4NetAppenderConfiguration.cs

```diff
@@ -49,7 +50,9 @@ public static void configure(string outputDirectory)
             Stream xmlConfigStream = assembly.get_manifest_stream(resource);
 
             XmlConfigurator.Configure(xmlConfigStream);
-            set_file_appender(outputDirectory);
+
+            if(outputDirectory != null)
+                set_file_appender(outputDirectory);

```

Comment:
yeah. even though I respectfully disagree with the style :) its nice when its cauht in the build.